Dockerfile

```
@@ -52,15 +52,15 @@ ARG TARGETPLATFORM
 # Skip puppeteer download during build (not needed for production)
 ENV PUPPETEER_SKIP_DOWNLOAD=true
 
-# Build assets and compile BirdNET-Go
+# Build assets and compile BirdNET-Go (non-embedded build)
 RUN --mount=type=cache,target=/go/pkg/mod,uid=10001,gid=10001 \
```

Hadolint (2.12.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)